src/models/Location.java

```diff
@@ -1,7 +1,47 @@
 package models;

```

Comment:
It may be easier to have the Location class just be an object to be sent as a JSON. This means the Location, Profile, and Review Model classes will just be objects without methods. Then, the LocationServelet, ProfileServelet, and ReviewServelets will deal with making the SQLCalls and constructing the objects to send to the frontend.